bullet/src/SDFFeatures.hh

```diff
@@ -25,6 +25,7 @@
 #include <gz/physics/sdf/ConstructModel.hh>
 #include <gz/physics/sdf/ConstructWorld.hh>
 #include <gz/physics/sdf/ConstructCollision.hh>
+#include <gz/physics/dartsim-plugin/Export.hh>
```

Comment:
This is the bullet implementation. Do we need Dartsim-plugin headers in here?

Reply:
uh that makes no sense, good catch. [8a87958](https://github.com/gazebosim/gz-physics/pull/711/commits/8a87958ac7503bd2c49ea0651ef1f484729dfdbf)